### PR TITLE
Add R versions 4.0.1 through 4.0.3

### PR DESCRIPTION
**Pull request by sjwmoveon**

### CODE REVIEW

File: plugins/R-build/share/R-build/4.0.1 [hunk 1]
```diff
@@ -0,0 +1,2 @@
+require_gcc
+install_package "R-4.0.0" "https://cloud.r-project.org/src/base/R-4/R-4.0.0.tar.gz#95fe24a4d8d8f8f888460c8f5fe4311cec656e7a1722d233218bc03861bc6f32" ldflags_dirs enable_memory_profiling enable_R_shlib with_blas with_lapack standard
```
Comment (dskard) [inline plugins/R-build/share/R-build/4.0.1]: can you change tar file being downloaded from R-4.0.0.tar.gz to R-4.0.1.tar.gz

File: plugins/R-build/share/R-build/4.0.2 [hunk 2]
```diff
@@ -0,0 +1,2 @@
+require_gcc
+install_package "R-4.0.2" "https://cloud.r-project.org/src/base/R-4/R-4.0.0.tar.gz#d3bceab364da0876625e4097808b42512395fdf41292f4915ab1fd257c1bbe75" ldflags_dirs enable_memory_profiling enable_R_shlib with_blas with_lapack standard
```
Comment (dskard) [inline plugins/R-build/share/R-build/4.0.2]: can you change tar file being downloaded from R-4.0.0.tar.gz to R-4.0.2.tar.gz

File: plugins/R-build/share/R-build/4.0.3 [hunk 3]
```diff
@@ -0,0 +1,2 @@
+require_gcc
+install_package "R-4.0.3" "https://cloud.r-project.org/src/base/R-4/R-4.0.0.tar.gz#09983a8a78d5fb6bc45d27b1c55f9ba5265f78fa54a55c13ae691f87c5bb9e0d" ldflags_dirs enable_memory_profiling enable_R_shlib with_blas with_lapack standard
```
Comment (dskard) [inline plugins/R-build/share/R-build/4.0.3]: can you change tar file being downloaded from R-4.0.0.tar.gz to R-4.0.3.tar.gz

**Review by dskard — CHANGES_REQUESTED** [review]
Ii appreciate you adding these newer versions of R to the repo. Can you update the names of the R tar files being downloaded so they match the SHA256 and version of R that is expected to be installed.

File: plugins/R-build/share/R-build/4.0.3 (hunk 3 above)
Comment (sjwmoveon) [reply plugins/R-build/share/R-build/4.0.3]: Done for all 3.  Thanks for catching that!

**Comment by sjwmoveon** [comment]
Updated the versions, thanks for catching that!

**Comment by jcrodriguez1989** [comment]
thanks for your contribution!
I guess that creating a GitHub Actions script that updates the versions would be great :)
Let me check the PR, and I will merge it!

**Comment by jcrodriguez1989** [comment]
Thanks for your contribution!!

**Pull request merged**